pubspec.yaml

```diff
@@ -7,6 +7,9 @@ environment:
   sdk: ">=2.16.0 <3.0.0"
   flutter: 2.10.5
 
+dependency_overrides: 
```

Comment:
flame 1.2.0 has Class `ParentIsA`  which is defined in packages/pinball_flame/lib/src/parent_is_a.dart

So if upgrade flame version, you should concurrently fix the `ParentIsA`  class reference.

Reply:
Yeah, I understood.

But here I mean specifically about the dependency override attribute.

Since you remove the ^ from the other places I believe this attribute here is not necessary

Reply:
OK, I understand!